Does not build cleanly on Mac OS/arm64 tho:

Undefined symbols for architecture arm64:
  "FileWatcher::clear()", referenced from:
      LaunchHatari(LaunchSettings const&, Session const*) in launcher.o
  "FileWatcher::addPath(QString const&)", referenced from:
      LaunchHatari(LaunchSettings const&, Session const*) in launcher.o
  "FileWatcher::addPaths(QStringList const&)", referenced from:
      LaunchHatari(LaunchSettings const&, Session const*) in launcher.o
  "FileWatcher::FileWatcher(Session const*)", referenced from:
      Session::createFileWatcherInstance() in session.o
ld: symbol(s) not found for architecture arm64

--- a/tools/hrdb/hrdb.pro
+++ b/tools/hrdb/hrdb.pro
@@ -35,6 +35,7 @@ SOURCES += \
     models/symboltable.cpp \
     models/symboltablemodel.cpp \
     models/targetmodel.cpp \
+    models/filewatcher.cpp \
     transport/dispatcher.cpp \
     ui/addbreakpointdialog.cpp \
     ui/breakpointswidget.cpp \
@@ -73,6 +74,7 @@ HEADERS += \
     models/symboltable.h \
     models/symboltablemodel.h \
     models/targetmodel.h \
+    models/filewatcher.h \
     transport/dispatcher.h \
     transport/remotecommand.h \
     ui/addbreakpointdialog.h \

